monitor/index.js

```diff
@@ -1,114 +1,60 @@
-        results.ok = results.total.length === 0
-        res.json(results)
-      } catch (e) {
-        next(e)
-      }
-    })
+    // this will eventually be handled by your error handling middleware
+    next(e)
   }
-}
+})
 
-app.get('/', async (req, res, next) => {
+bridgeRouter.get('/validators', async (req, res, next) => {
   try {
-    const results = await readFile('./responses/getBalances.json')
+    const results = await readFile(`./responses/${req.params.bridgeName}/validators.json`)
     res.json(results)
   } catch (e) {
     // this will eventually be handled by your error handling middleware
     next(e)
   }
 })
 
-app.get('/validators', async (req, res, next) => {
+bridgeRouter.get('/eventsStats', async (req, res, next) => {
   try {
-    const results = await readFile('./responses/validators.json')
-    results.homeOk = true
-    results.foreignOk = true
-
-    if (MONITOR_VALIDATOR_HOME_TX_LIMIT) {
-      for (const hv in results.home.validators) {
-        if (results.home.validators[hv].leftTx < MONITOR_TX_NUMBER_THRESHOLD) {
-          results.homeOk = false
-          break
-        }
-      }
-    }
-
-    if (MONITOR_VALIDATOR_FOREIGN_TX_LIMIT) {
-      for (const hv in results.foreign.validators) {
-        if (results.foreign.validators[hv].leftTx < MONITOR_TX_NUMBER_THRESHOLD) {
-          results.foreignOk = false
-          break
-        }
-      }
-    }
-
-    results.ok = results.homeOk && results.foreignOk
+    const results = await readFile(`./responses/${req.params.bridgeName}/eventsStats.json`)
     res.json(results)
   } catch (e) {
     // this will eventually be handled by your error handling middleware
     next(e)
   }
 })
 
-// responses/eventsStats.json
-app.get('/eventsStats', async (req, res, next) => {
+bridgeRouter.get('/alerts', async (req, res, next) => {
   try {
-    const results = await readFile('./responses/eventsStats.json')
-    results.ok =
-      (results.onlyInHomeDeposits || results.home.deliveredMsgNotProcessedInForeign).length === 0 &&
-      (results.onlyInForeignDeposits || results.home.processedMsgNotDeliveredInForeign).length === 0 &&
-      (results.onlyInHomeWithdrawals || results.foreign.deliveredMsgNotProcessedInHome).length === 0 &&
-      (results.onlyInForeignWithdrawals || results.foreign.processedMsgNotDeliveredInHome).length === 0
+    const results = await readFile(`./responses/${req.params.bridgeName}/alerts.json`)
     res.json(results)
   } catch (e) {
-    // this will eventually be handled by your error handling middleware
     next(e)
   }
 })
 
-app.get('/alerts', async (req, res, next) => {
+bridgeRouter.get('/stuckTransfers', async (req, res, next) => {
   try {
-    const results = await readFile('./responses/alerts.json')
-    results.ok = !results.executeAffirmations.mostRecentTxHash && !results.executeSignatures.mostRecentTxHash
+    const results = await readFile(`./responses/${req.params.bridgeName}/stuckTransfers.json`)
```

Comment:
It should be provided only in case of `isV1Bridge()`

Reply:
Now that `index.js` doesn't have any information about the bridges, we cannot detect if there is a `V1bridge` at the moment of creating the endpoints. Should we create a custom json response for this case if the  `stuckTransfers.json` file is not present?

Reply:
Ok. I see the message we are discussing above should be enough